src/drivers/battery/bq40z50/bq40z50.h

```diff
@@ -116,10 +132,10 @@ int manufacturer_name();
  */
 int serial_number();
 
-extern device::Device *BATT_SMBUS_I2C_interface(int bus);
-typedef device::Device *(*BATT_SMBUS_constructor)(int);
+extern device::Device *BQ40Z50_I2C_interface(int bus);
+typedef device::Device *(*BQ40Z50_constructor)(int);
 
-class BATT_SMBUS : public device::CDev
+class BQ40Z50 : public device::CDev
```

Comment:
This doesn't actually need to be a CDev, although it's fine to keep it this way for now if you want.